apps/emqx_connector/src/emqx_connector_resource.erl

```diff
@@ -378,3 +378,13 @@ override_start_after_created(Config, Opts) ->
 
 set_no_buffer_workers(Opts) ->
     Opts#{spawn_buffer_workers => false}.
+
+%% TODO: introduce a formal callback?
+redact(Conf, Type) when
+    Type =:= http;
```

Comment:
haven't we decided to use atom or binary?

Reply:
I've added both as safety.  It's used in multiple places (directly or indirectly) that could not follow a clean convention...